### PR TITLE
Add all useful fmtc dithertypes and set default to void

### PR DESCRIPTION
**Pull request by Vodes**

### CODE REVIEW

**Comment by Vodes** [comment]
This more or less supercedes #84 but discussion about the default is still very welcome.

File: vstools/functions/utils.py [hunk 1]
```diff
@@ -54,6 +54,49 @@ class DitherType(CustomStrEnum):
...
+    OSTROMOUKHOV = 'ostromoukhov'
+    """Another error diffusion kernel. Slow, available only for integer input at the moment. Avoids usual F-S artefacts."""
+
+    VOID = 'void'
+    """A way to generate blue-noise dither and has a much better visual aspect than ordered dithering."""
+
+    QUASIRANDOM = 'quasirandom'
+    """Dither using quasirandom sequences. Good intermediated between Void and cluster and error diffusion algorithms."""
+
+    def apply(self, clip: vs.VideoNode, format: vs.VideoFormat, range_in: ColorRange, range_out: ColorRange, format_in: vs.VideoFormat) -> vs.VideoNode:
```
Comment (Setsugennoao) [inline vstools/functions/utils.py:78]: Format overrides a python builtin, you should rename both to ftm_in and ftm_out, following the range naming scheme, and maybe reorder as fmt in,out range in,out.
Also do we really need format_in for checking the vs.FLOAT? Can't you just see from clip.format.sample_type

File: vstools/functions/utils.py [hunk 2]
```diff
@@ -54,6 +54,49 @@ class DitherType(CustomStrEnum):
...
+            return clip.fmtc.bitdepth(dmode=DitherType._fmtc_types().get(self.value), bits=format.bits_per_sample, 
+                                      fulls=None if not range_in else range_in == ColorRange.FULL, 
+                                      fulld=None if not range_out else range_out == ColorRange.FULL)
+    @property
+    def _is_fmtc(self) -> bool:
+        return self.value in DitherType._fmtc_types().keys()
+
+    @staticmethod
+    def _fmtc_types() -> dict[DitherType, int]:
+        return {DitherType.SIERRA_LITE: 3, DitherType.STUCKI: 4, DitherType.ATKINSON: 5, DitherType.ERROR_DIFFUSION_FMTC: 6,
```
[the remark this answers is not in the document]
Comment (Vodes) [reply vstools/functions/utils.py:97]: wasn't sure if this was possible in enums but sure I'll change that

**Review by Vodes — COMMENTED** [review]
lgtm

**Pull request merged**